### PR TITLE
fix QR code for Receive Funds; Add Unit token to be specific based on…

### PR DESCRIPTION
**Pull request by wirednkod**
… chain

Solves #69

### CODE REVIEW

File: projects/burnr/package-lock.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      "dev": true
+    },
+    "csstype": {
+      "version": "3.0.6",
+      "resolved": "https://registry.npmjs.org/csstype/-/csstype-3.0.6.tgz",
+      "integrity": "sha512-+ZAmfyWMT7TiIlzdqJgjMb7S4f1beorDbWbsocyK4RaiqA5RTX3K14bnBWmmA9QEM0gRdsjyyrEmcyga8Zsxmw==",
+      "dev": true
+    }
+  }
+}
```
Comment (raoulmillais) [inline projects/burnr/package-lock.json:43]: Accidentaly commited package-lock.json

File: projects/burnr/src/components/InputFunds.tsx [hunk 2]
```diff
@@ -5,17 +5,19 @@ import { Button, Grid, TextField, Box, InputAdornment } from '@material-ui/core'
...
 
-const InputFunds: React.FunctionComponent<Props> = ({ total, currency }: Props) => {
+const InputFunds: React.FunctionComponent<Props> = ({ total, currency, hidePercentages = false }: Props) => {
 	const [value, setValue] = React.useState<number | ''>('');
 	const handleChangeButton = (e: MouseEvent) => {
 		setValue(parseInt((e.currentTarget as HTMLButtonElement).value) * total);
 		document.getElementById('SendFundsAmountField')!.focus();
 	};
 	const handleChange = (e: ChangeEvent) => {
+		console.log('e', e)
```
Comment (raoulmillais) [inline projects/burnr/src/components/InputFunds.tsx:20]: Leftover console.log

File: projects/burnr/src/components/ReceiveFundsForm.tsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+			direction='column'
+			className={classes.container}
+		>
+			{/*<Grid item>
+				<InputFunds
+                    hidePercentages
+					total={100}
+					currency={ballanceArr[3]}
+				/>
+            </Grid>*/}
```
Comment (raoulmillais) [inline projects/burnr/src/components/ReceiveFundsForm.tsx:52]: Did you mean to leave this commented out code in?  Looks like you have an actual implementation below so its confusing.  I prefer not to keep commented code in (without an explanation fir it)... you can always look back through the git history if you want to recover something that was there in the past.

File: projects/burnr/src/components/SendFundsForm.tsx [hunk 4]
```diff
@@ -1,10 +1,14 @@
...
+import { AccountContext } from '../utils/contexts';
 
 import { InputAddress, InputFunds } from '../components';
 import { makeStyles, createStyles, Theme, Grid, Button } from '@material-ui/core';
+import { useBalance } from '../hooks'
 
 const useStyles = makeStyles((theme: Theme) =>
-	createStyles({
+	{
+		console.log(theme)
```
Comment (raoulmillais) [inline projects/burnr/src/components/SendFundsForm.tsx:10]: Leftover console.log

File: projects/burnr/src/components/SendFundsForm.tsx [hunk 5]
```diff
@@ -19,11 +23,16 @@ const useStyles = makeStyles((theme: Theme) =>
...
 			},
 		},
-	})
+	})}
 );
 
 const SendFundsForm: React.FunctionComponent = () => {
 	const classes = useStyles();
+	console.log('classes0, 0', classes)
+    const { account } = useContext(AccountContext);
```
Comment (raoulmillais) [inline projects/burnr/src/components/SendFundsForm.tsx:32]: Indentation is off here and there's another console.log

**Review by raoulmillais — APPROVED** [review]
This looks good apart from my minor nitpicks :+1: I've checked it out and run it.

File: projects/burnr/src/components/ReceiveFundsForm.tsx (hunk 3 above)
Comment (goldsteinsveta) [reply projects/burnr/src/components/ReceiveFundsForm.tsx:52]: There are also unused styles. I don't think there's need to keep the commented lines.

```js
import React, { useContext } from 'react';

import { AccountContext } from '../utils/contexts';

import QRCode from 'qrcode.react';
import { makeStyles, createStyles, Theme, Grid } from '@material-ui/core';

const useStyles = makeStyles((theme: Theme) =>
	createStyles({
		qrBox: {
			alignContent: 'center',
			justifyContent: 'center',
			paddingTop: theme.spacing(3)
		},
	})
);
  

const ReceiveFundsForm: React.FunctionComponent = () => {
	const { account } = useContext(AccountContext);
	const classes = useStyles();
	return (
		<Grid
			container
			className={classes.qrBox}
		>
			<Grid item>
				<QRCode
					value={account.userAddress}
					size={400}
					bgColor={"#ffffff"}
					fgColor={"#000000"}
					includeMargin={true}
					renderAs={"svg"}
				/>
			</Grid>
		</Grid>
	);
};

export default ReceiveFundsForm;
```

**Review by goldsteinsveta — APPROVED** [review]

File: projects/burnr/src/components/ReceiveFundsForm.tsx (hunk 3 above)
Comment (wirednkod) [reply projects/burnr/src/components/ReceiveFundsForm.tsx:52]: > Did you mean to leave this commented out code in? Looks like you have an actual implementation below so its confusing. I prefer not to keep commented code in (without an explanation fir it)... you can always look back through the git history if you want to recover something that was there in the past.

I was not sure if the input should stay there or not. I'll remove it and retrieve it from history if needed.

**Pull request merged**